recipes/cern-root/all/conanfile.py

```diff
@@ -21,7 +21,9 @@ class PythonOption:
 
 class CernRootConan(ConanFile):
     name = "cern-root"
-    version = "v6-22-02"
+    # version format is intentional, ROOT does not follow strict SemVer.
+    # see: https://root.cern/about/versioning/
+    version = "v6-22-06"
```

Comment:
```suggestion
```

You can specify the version from the command line with `conan create conanfile.py v6-22-06@`